src/lib/Preview.js

```
@@ -879,7 +897,9 @@ class Preview extends EventEmitter {
             });
 
             // If there wasn't an error, use Events API to log a preview
-            this.logPreviewEvent(this.file.id, this.options);
+            if (!this.options.skipServerUpdate) {
+                this.logPreviewEvent(this.file.id, this.options);
+            }
```

Yeah lets remove this `if` check. And just let the request fail it was offline.